src/libcollections/vec.rs

```diff
@@ -1296,6 +1296,16 @@ impl<T: PartialEq> Vec<T> {
     pub fn dedup(&mut self) {
         self.dedup_by(|a, b| a == b)
     }
+
+    #[unstable(feature = "vec_remove_item", reason = "recently added", issue = "38143")]
```

Comment:
This should have a doc comment clarifying that it only removes the first item found.